devices.js

```diff
@@ -15490,6 +15490,29 @@ const devices = [
         extend: generic.light_onoff_brightness_colortemp,
     },
 
+    // Atsmart
+    {
+        zigbeeModel: ['Z601', 'Z602', 'Z603', 'Z604'],
```

Comment:
Are these 1/2/3/4 gang versions? If yes please add them separately

Reply:
no，all these models are the same device

Reply:
Why is the try/catch here: https://github.com/Koenkk/zigbee-herdsman-converters/pull/1880/files#diff-5520dd49755e0a353d3857b8963a91cb4b8d87a01b6b8eb7bb6cce1c180eac83R15511 ? // dip switch for 1-3 gang -> sounds like there are 1 till 3 gang versions?

Reply:
this switch has a dip-switch under its panel, this dip-switch changes the model and endpoints, so i combined the code
![1424124595-18](https://user-images.githubusercontent.com/580122/101868498-6fc2f100-3bb8-11eb-945a-1ab4aa3f7ef7.jpg)

Reply:
Thanks